src/components/layout/GenericLayout/NavTools/index.tsx

```diff
@@ -0,0 +1,117 @@
+import React from 'react'
+import styled from 'styled-components'
+
+import { MEDIA } from 'const'
+import { NavStyled } from '../Menu'
+import { Frame } from 'components/common/Frame'
+
+import SettingsImage from 'assets/img/settings.svg'
+import SettingsImageWhite from 'assets/img/settings-white.svg'
+import NotificationsImage from 'assets/img/bell.svg'
+import NotificationsImageWhite from 'assets/img/bell-white.svg'
+
+const SettingsLink = styled.li`
+  margin: 0 2.4rem 0 0;
+  order: 3;
+
+  > a {
+    display: block;
+    margin: 0;
+    width: 3.6rem;
+    height: 1.4rem;
+    content: '';
+    background: url(${SettingsImage}) no-repeat center/contain;
+  }
+
+  &:hover > a {
+    background: url(${SettingsImageWhite}) no-repeat center/contain;
+  }
+`
+
+const NotificationsLink = styled.li`
+  order: 4;
+
+  > a {
+    display: block;
+    margin: 0;
+    width: 3.6rem;
+    height: 1.4rem;
+    content: '';
+    background: url(${NotificationsImage}) no-repeat center/contain;
+  }
+
+  &:hover > a {
+    background: url(${NotificationsImageWhite}) no-repeat center/contain;
+  }
+`
+
+const Wrapper = styled(NavStyled)`
+  margin: 0 0 0 auto;
+  padding: 0;
+
+  @media ${MEDIA.mediumDown} {
+    flex-direction: row;
+    justify-content: flex-start;
+    justify-self: center;
+    padding: 1rem;
+    position: fixed;
+    bottom: 0;
+    left: 0;
+    width: 100%;
+    z-index: 99;
+    height: 7.2rem;
+    border-radius: 0;
+    background-color: var(--color-gradient-2);
+  }
+
+  > li {
+    display: flex;
+    align-items: center;
+
+    &:not(:last-of-type)::after {
+      content: '';
+      display: block;
+      width: 0.1rem;
+      height: 2.4rem;
+      background: var(--color-border);
+      margin: 0 1.6rem;
+      position: absolute;
+      right: -2.8rem;
+    }
+
+    @media ${MEDIA.mediumDown} {
```

Comment:
you could move this rule into the already present @media above. For me it's easier to read when it's all in the same media query BUT i do agree it's easier to code it the way it is now. Up to you

Reply:
Yeah, in terms of 'grouping' all rules for a specific selector, I always opt to keep the media queries closely aligned (nested) for the element it applies to. Keeps the code dryer in terms of not having to replicate writing the selector. Thoughts on this?

Reply:
Agreed @biocom but it can be easy to miss things when changing media query code later imo

Reply:
Can you name a scenario where you'd change a media query somewhere, and that impacting a specific media query nested in particular element?

Reply:
sure, the size of the media query

Reply:
The actual value of the media query is handled in the global variables file. Globally changing the variable name in all files can be an easy search 'n replace. Or you mean something else?

Reply:
If I get this right, I would lean to Michel approach, where the CSS of one component is close to the functional component it applies. I don't have a strong opinion, in this case, it wouldn't hurt my eyes to have the `ol` and `li` media defined close to each other (since the css is small, and is easy to maintain either way)

Reply:
I'll merge this PR as it is, but let's decide and agree what is best